cns/Dockerfile

```diff
@@ -28,6 +28,9 @@ FROM mcr.microsoft.com/windows/servercore@sha256:6fdf140282a2f809dae9b13fe441635
 # intermediate for win-ltsc2022
 FROM mcr.microsoft.com/windows/servercore@sha256:45952938708fbde6ec0b5b94de68bcdec3f8c838be018536b1e9e5bd95e6b943 as ltsc2022
 
+# intermediate for win2025
+FROM mcr.microsoft.com/windows/servercore:ltsc2022-KB5035857-amd64 as win2025
```

Comment:
can we use a sha digest here instead of a tag?

Reply:
it's we do not have official ws2025 image and we use this temporarily to build win2025 binaries/docker images, so can we will use sha in the next release